config_builder.go

```diff
@@ -686,15 +686,15 @@ func (d *DefaultWalletImpl) BuildChainControl(
 	// Create, and start the lnwallet, which handles the core payment
 	// channel logic, and exposes control via proxy state machines.
 	lnWalletConfig := lnwallet.Config{
-		Database:           partialChainControl.Cfg.ChanStateDB,
-		Notifier:           partialChainControl.ChainNotifier,
-		WalletController:   walletController,
-		Signer:             walletController,
-		FeeEstimator:       partialChainControl.FeeEstimator,
-		SecretKeyRing:      keyRing,
-		ChainIO:            walletController,
-		DefaultConstraints: partialChainControl.ChannelConstraints,
-		NetParams:          *walletConfig.NetParams,
+		Database:         partialChainControl.Cfg.ChanStateDB,
+		Notifier:         partialChainControl.ChainNotifier,
+		WalletController: walletController,
+		Signer:           walletController,
+		FeeEstimator:     partialChainControl.FeeEstimator,
+		SecretKeyRing:    keyRing,
+		ChainIO:          walletController,
+		DefaultDustLimit: lnwallet.DustLimitUnknownWitness(),
```

Comment:
Does the unknown case need to be handled? 

The dust limit (tho it's super confusing) is also a standard/static value in most uses.

Reply:
This was done because it was the underlying implementation and I was trimming the values. In this case We can try and rip it out entirely and see where that bottoms out.

Reply:
I ended up ripping it out of the config and essentially inlining the `DustLimitUnknownWitness` at the reference sites. You can take a look and see if you like it, but it looks like our unit test suite is fragile in this area wrt this value getting propagated to the right places.